app/layout.tsx

```diff
@@ -12,18 +13,38 @@ type LayoutProps = {
   children: React.ReactNode,
 };
 
-const sourceSans = Source_Sans_3({
+const druk = localFont({
+  src: '../public/fonts/Druk-Super-Web.woff2',
+  weight: '900',
+  variable: '--font-druk',
+});
+
+const druk_wide = localFont({
+  src: '../public/fonts/DrukWide-Bold-Web.woff2',
+  weight: '700',
+  variable: '--font-druk-wide',
+});
+
+const source_sans = Source_Sans_3({
   subsets: ['latin'],
+  style: ['italic','normal'],
   display: 'swap',
   variable: '--font-source-sans',
 });
 
-const sourceSerif = Source_Serif_4({
+const source_serif = Source_Serif_4({
   subsets: ['latin'],
+  style: ['italic','normal'],
   display: 'swap',
   variable: '--font-source-serif',
 });
 
+const stanford = localFont({
+  src: '../public/fonts/stanford.woff2',
```

Comment:
Keeping only the woff2 version of the Stanford font. I think we don't need the truetype anymore
https://caniuse.com/woff2